frontend/javascripts/router.js

```diff
@@ -368,7 +368,7 @@ class ReactRouter extends React.Component<Props> {
               <SecuredRoute
                 isAuthenticated={isAuthenticated}
                 path="/reports/timetracking"
-                component={TimeLineView}
+                render={() => <TimeLineView />}
```

Comment:
This actually fixes the weird flow bug/behaviour, This is also used some lines above :slightly_smiling_face: